genesis-tools/genesis-populate/src/lib.rs

```diff
@@ -19,11 +19,15 @@ use near_primitives::types::{AccountId, Balance, EpochId, ShardId, StateChangeCa
 use near_store::{get_account, set_access_key, set_account, set_code, Store, TrieUpdate};
 use nearcore::{NearConfig, NightshadeRuntime};
 use std::collections::BTreeMap;
+use std::hash::{Hash, Hasher};
 use std::path::{Path, PathBuf};
 use std::sync::Arc;
 
-fn get_account_id(account_index: u64) -> AccountId {
-    AccountId::try_from(format!("near_{}_{}", account_index, account_index)).unwrap()
```

Comment:
Yeah it's a bit weird. It's to make it longer I suppose, as the account name length does matter quite a bit for how deep in a trie it can be.

Not sure if it was intentional by whomever designed this but this creates somewhat interesting paths for every account that look something like this:
- extension node from root to "near_"
- several branches to get through first account index
- another extension for the second index

Almost the same thing could be achieved with "near_{account_index}_blablabla". But now there is a potential of sharing the extension node, as the state will be the same for all accounts and I think they will produce the same hashes.

Honestly, I don't think it matters. But let's not swap out too many gears at once.